modules/openapi-generator/src/main/resources/Java/modelEnum.mustache

```diff
@@ -51,6 +51,7 @@ import com.google.gson.stream.JsonWriter;
     return String.valueOf(value);
   }
 
+{{#jackson}}
 {{#jackson}}
```

Comment:
I think it's weird to open #jackson twice. Can you just extend the one for JsonCreator?